tests/processor/test_orchestrator.py

```diff
@@ -48,7 +50,25 @@ class OrchestratorTest(MasuTestCase):
     """Test Cases for the Orchestrator object."""
 
     def setUp(self):
-        pass
+        self.mock_accounts = []
+        account1 = CostUsageReportAccount('cred1',
+                                          'billingsource1',
+                                          'customer1',
+                                          'AWS',
+                                          'customer1schema')
+        account2 = CostUsageReportAccount('cred2',
+                                          'billingsource2',
+                                          'customer2',
+                                          'AWS',
+                                          'customer2schema')
+        account3 = CostUsageReportAccount('cred3',
+                                          'billingsource3',
+                                          'customer3',
+                                          'AWS',
+                                          'customer3schema')
+        self.mock_accounts.append(account1)
+        self.mock_accounts.append(account2)
+        self.mock_accounts.append(account3)
```

Comment:
This can be more DRY just directly assigning to the mock_accounts array. Or, you can just use `extend()` to append multiple items instead of having three uses of `append()`

Either option is fine by me.

Reply:
ok